### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by KOLANICH**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,7 @@
+root = true
```
Comment (dalemyers) [inline .editorconfig]: Can you remove this file? It's a tiny project. I don't want to start enforcing things like this. I'd rather use a linter to enforce these sorts of things.

File: csvLaTeXTable.py [hunk 2]
```diff
@@ -105,57 +105,57 @@ def parse(self,filename):
...
+        options.table_spec = "l"
+        
+    #If a filename isn't specified then attempt to take the last argument
+    if(options.filename == ""):
+        filename = sys.argv[len(sys.argv)-1]
+    else:
+        filename = options.filename
+
+    if not os.path.isfile(filename):
+        print(filename,"is not a valid file name. The file name should be specified with the '-f' option, or be the last argument.")
```
Comment (dalemyers) [inline csvLaTeXTable.py]: Lets ensure it works with Python 2 and 3. Add a `from __future__ import print_function` at the top (and any other required changes).

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+name = csvLaTeXTable
+author = Dale Myers
+url = https://github.com/dalemyers/CSV-to-LaTeX-Table
+description = Convert your CSV/TSV files to a latex table
+long_description = file: README.md
+keywords = csv, tsv, TeX, LaTeX, table
+license = MIT
+classifiers =
+	Programming Language :: Python
+	Programming Language :: Python :: 3
```
Comment (dalemyers) [inline setup.cfg]: Add Python 2 here.

File: setup.cfg [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	License :: OSI Apprived :: MIT
+	Operating System :: OS Independent
+
+[options]
+zip_safe = True
+include_package_data = True
+py_modules = csvLaTeXTable
+
+[options.entry_points]
+console_scripts = csvLaTeXTable = csvLaTeXTable:main
```
Comment (dalemyers) [inline setup.cfg]: This is pretty cool. I didn't actually know you could do this. 👍

File: csvLaTeXTable.py [hunk 5]
```diff
@@ -105,57 +105,57 @@ def parse(self,filename):
 
         self.print_table_ending()
 
+def main():
```
Comment (dalemyers) [inline csvLaTeXTable.py]: Can we rename this to something like `_handle_arguments()` while we are here? Main isn't particularly helpful and I'd rather it not be listed if anyone wants to use this as a module.

File: .editorconfig (hunk 1 above)
Comment (KOLANICH) [reply .editorconfig]: >Can you remove this file?

I insist on that file. It must be in any project: some editors (more concrete - the one I use) unless configured in this file insert tabs.

File: csvLaTeXTable.py (hunk 5 above)
Comment (KOLANICH) [reply csvLaTeXTable.py]: It is not for handling arguments, it is purely for entry point.

File: setup.cfg (hunk 3 above)
Comment (KOLANICH) [reply setup.cfg]: done

File: csvLaTeXTable.py (hunk 2 above)
Comment (KOLANICH) [reply csvLaTeXTable.py]: added.

**Comment by dalemyers** [comment]
Going to approve since I just don't have time to do proper PRs for this.

**Pull request merged**